src/framework/components/scroll-view/component.js

```diff
@@ -59,6 +59,9 @@ Object.assign(pc, function () {
         this._scroll = new pc.Vec2();
         this._velocity = new pc.Vec3();
 
+        this._disabledContent = false;
```

Comment:
[nit] `_disabledContentInput` would be a bit clearer – we're not actually disabling the elements.